### PR TITLE
chore: Use narrow layout container for content pages

### PR DESCRIPTION
**Pull request by jasonlcrane**
This addresses #452, updating to use the narrow container for content pages on the site. I also felt like it needed some additional space between the header and start of the content on larger screens, but I defer to @beep and the design team.

Feedback welcome, especially since this is my first time poking my head into React.

### CODE REVIEW

File: src/templates/content.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const ContentPage = ({ data }) => {
     <Layout
       title={page.title}
       navigation={page.navigationGroup ? page.navigationGroup.pages : false}
+      narrow="true"
```
Comment (kevee) [inline src/templates/content.js:11]: Props that are truthy should just be empty, so just change `narrow="true"` to `narrow`

```javascript
<Layout [stuff] narrow>
```

File: src/components/common/container.module.scss [hunk 2]
```diff
@@ -17,5 +17,8 @@
   }
   &.container-narrow {
     max-width: 900px;
+    @media (min-width: $viewport-lg) {
```
Comment (kevee) [inline src/components/common/container.module.scss:20]: I think we should leave the margin where it is.

File: src/templates/content.js (hunk 1 above)
Comment (jasonlcrane) [reply src/templates/content.js]: Thanks, makes sense. Updated.

File: src/components/common/container.module.scss (hunk 2 above)
Comment (jasonlcrane) [reply src/components/common/container.module.scss]: I updated to remove this.

**Comment by jasonlcrane** [comment]
Thanks for the feedback, Kevin! Those fixes are made.

**Comment by kissane** [comment]
Thanks very much for this @jasonlcrane! @beep when you have time next week could you give this a look for us? I like the width of the content much better, but I feel like our grid needs some design braining here.

**Comment by schwartzadev** [comment]
Photos, for reference:

**current**:
image



**proposed**:
image

**Comment by kevee** [comment]
I'm going to hold off for design review.

**Comment by beep** [comment]
> @beep when you have time next week could you give this a look for us? I like the width of the content much better, but I feel like our grid needs some design braining here.

Absolutely, @kissane! I’ll take a look tomorrow morning.

File: src/components/common/container.module.scss (hunk 2 above)
Comment (kevee) [reply src/components/common/container.module.scss]: Thanks!

File: src/templates/content.js (hunk 1 above)
Comment (kevee) [reply src/templates/content.js]: 🙌

**Comment by beep** [comment]
Hey folks! Just poking around a bit at this, and I think this is looking really promising. My main feedback is that the narrow layout probably shouldn’t be centered; ideally, we’d preserve that left-edge line, and pull the `max-width` in a bit. Here’s a quick in-browser mockup to show what I mean:

image

(This is with a `max-width` of 42rem, or ~670px. Something a bit narrower than 900px feels a bit more comfortable to me.)

What do you think? Does that seem doable? Happy to help with this, if I can.

**Comment by jasonlcrane** [comment]
Great call to adjust the alignment and width @beep. I pushed up a new change to update for that. This changes the HTML structure slightly (adds a second container div) when using a narrow width, and will result in the narrow container not being able to be used to center narrow content on a page, at least as it is structured currently. That may have been the original intent, but it is not used for that purpose anywhere on the site that I am able to find. I updated the storybook to reflect this change as well.

File: src/components/common/container.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
-      containerStyles.narrow}`}
-  >
-    {children}
-  </div>
-)
+export default ({ children, narrow }) => {
+  if (narrow) {
+    return (
+      <div className={`container ${containerStyles.container}`}>
+        <div className={`${narrow && containerStyles.containerNarrow}`}>
```
Comment (kevee) [inline src/components/common/container.js:8]: This is within an if statement for narrow = true, so you can remove the `narrow &&` part.

Comment (kevee) [reply src/components/common/container.js:8]: Actually, I think you can just use this and not wrap it in an if statement:

```
<div className={`container ${containerStyles.container} ${narrow && conatinerStyles.containerNarrow}`}>
```

Then change the SCSS module to:

```scss
&.container-narrow {. <-- remove the space
```

The extra class will make it more specific than the general container component.

Comment (jasonlcrane) [reply src/components/common/container.js:8]: Thanks @kevee. I'll remove the redundant `narrow &&` part. 

On your second comment, take a look at the deploy preview (https://deploy-preview-474--upbeat-lovelace-3e9fff.netlify.com/about-project). I made `container-narrow` a child of the primary container so it stays on the page's  grid (keeping the content even with the page title). I may be missing a better solution, but applying the narrow style to the first container div centers it on the page, which is essentially the original change.

Comment (kevee) [reply src/components/common/container.js:8]: Ah gotcha, I'm fine with that.

**Comment by kissane** [comment]
The only thing I see missing here, [looking at @beep's comp](https://github.com/COVID19Tracking/website/issues/452), is additional space between the header and the body content. Super psyched about this! Getting it just in time for our blog machine to pick it up.

**Comment by jasonlcrane** [comment]
Thanks @kissane! I updated to add some additional space below the header (it's the same as above the footer, now). I see that @schwartzadev is planning to pick up the other changes from @beep's comp on #452, so I'm wondering if we should merge this and those changes together or let this one stand separately.

**Comment by schwartzadev** [comment]
I'm flexible -- I'll tackle those changes tomorrow, if that makes any difference.

I would suggest keeping them separate, since it makes my git side of things a smidge easier, but I'm pretty agnostic.

Unless I hear otherwise before tomorrow, I'll plan on using a separate branch -- we can always merge the changes together before merging to master :+1:

**Comment by beep** [comment]
If https://deploy-preview-474--upbeat-lovelace-3e9fff.netlify.com/about-project is the latest version, I think it’s a.) lookin’ _grand_ and b.) if @schwartzadev is willing to tackle further refinements separately, then we could get this shipped out the door!

**Comment by schwartzadev** [comment]
@jasonlcrane I'm going to tap you to approve 0573f96738432f3ce065b2682b656e4e18d52af8 when you get a chance :+1:

**Comment by jasonlcrane** [comment]
Nice improvement. Looks good to me 👍

**Pull request merged**